src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback, useState } from 'react';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
+import { invert } from 'lodash';
```

Comment:
Could you please avoid that `invert`? Previous `defaultCloseOrderReasons.js` is useless in previous form.